otbenchmark/mini_Benchmark.py

```diff
@@ -0,0 +1,182 @@
+            p31,
+            p33,
+            p35,
+            p38,
+            p53,
+            p55,
+            p57,
+            p75,
+            p89,
+            p107,
+            p110,
+            p111,
+            p63,
+            p91,
+            p60,
+            p77,
+        ]
+        return Liste_problem
+
+    def FORM(Problem):
+        # Cobyla algorithm
+        myCobyla = ot.Cobyla()
+        # Resolution options:
+        eps = 1e-3
+        myCobyla.setMaximumEvaluationNumber(100)
+        myCobyla.setMaximumAbsoluteError(eps)
+        myCobyla.setMaximumRelativeError(eps)
+        myCobyla.setMaximumResidualError(eps)
+        myCobyla.setMaximumConstraintError(eps)
+        myEvent = Problem.getEvent()
+        inputVector = myEvent.getAntecedent()
+        myDistribution = inputVector.getDistribution()
+        g = Problem.getEvent().getFunction()
+        algoFORM = ot.FORM(myCobyla, myEvent, myDistribution.getMean())
+        initialNumberOfCall = g.getEvaluationCallsNumber()
+        algoFORM.run()
+        resultFORM = algoFORM.getResult()
+        numberOfFunctionEvaluationsFORM = (
+            g.getEvaluationCallsNumber() - initialNumberOfCall
+        )
+        return [resultFORM, numberOfFunctionEvaluationsFORM]
+
+    def SORM(Problem):
+        # Cobyla algorithm
+        myCobyla = ot.Cobyla()
+        # Resolution options:
+        eps = 1e-3
+        myCobyla.setMaximumEvaluationNumber(100)
+        myCobyla.setMaximumAbsoluteError(eps)
+        myCobyla.setMaximumRelativeError(eps)
+        myCobyla.setMaximumResidualError(eps)
+        myCobyla.setMaximumConstraintError(eps)
+        myEvent = Problem.getEvent()
+        inputVector = myEvent.getAntecedent()
+        myDistribution = inputVector.getDistribution()
+        g = Problem.getEvent().getFunction()
+        algoSORM = ot.SORM(myCobyla, myEvent, myDistribution.getMean())
+        initialNumberOfCall = g.getEvaluationCallsNumber()
+        algoSORM.run()
+        resultSORM = algoSORM.getResult()
+        numberOfFunctionEvaluationsSORM = (
+            g.getEvaluationCallsNumber() - initialNumberOfCall
+        )
+        return [resultSORM, numberOfFunctionEvaluationsSORM]
+
+    def MonteCarloSampling(Problem, NBS, cv):
+
+        myEvent = Problem.getEvent()
+        g = Problem.getEvent().getFunction()
+        experiment = ot.MonteCarloExperiment()
+        myMC = ot.ProbabilitySimulationAlgorithm(myEvent, experiment)
+        myMC.setMaximumOuterSampling(NBS)
+        myMC.setBlockSize(1)
+        myMC.setMaximumCoefficientOfVariation(cv)
+        initialNumberOfCall = g.getEvaluationCallsNumber()
+        myMC.run()
+        resultMC = myMC.getResult()
+        numberOfFunctionEvaluationsMonteCarlo = (
+            g.getEvaluationCallsNumber() - initialNumberOfCall
+        )
+        return [
+            resultMC,
+            numberOfFunctionEvaluationsMonteCarlo,
+            myMC.drawProbabilityConvergence(),
```

Comment:
This is the default graph, when the probability is unknown. You should, however, plot the log(absolute error) vs log(iteration number) graph because the exact probability is known here.